circle.yml

```diff
@@ -0,0 +1,68 @@
+machine:
+  environment:
+    # We need to set this variable to let Anaconda take precedence
+    PATH: "/home/ubuntu/miniconda/envs/circleenv/bin:/home/ubuntu/miniconda/bin:$PATH"
+    MNE_DATA: "/home/ubuntu/mne_data"
+
+dependencies:
+  cache_directories:
+    - "/home/ubuntu/miniconda"
+    - "/home/ubuntu/.mne"
+    - "/home/ubuntu/mne_data"
+  # Various dependencies
+  pre:
+    # Get a running Python
+    - cd ~
+    # Disable pyenv (no cleaner way provided by CircleCI as it prepends pyenv version to PATH)
+    - rm -rf ~/.pyenv
+    - rm -rf ~/virtualenvs
+    # Get Anaconda and conda-based requirements
+    - >
+      if [ ! -d "/home/ubuntu/miniconda" ]; then
+        echo "Setting up conda";
+        wget -q http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O ~/miniconda.sh;
+        chmod +x ~/miniconda.sh;
+        ~/miniconda.sh -b -p /home/ubuntu/miniconda;
+        conda update --yes --quiet conda;
+        conda create -n circleenv --yes pip python=2.7 pip numpy scipy scikit-learn mayavi matplotlib sphinx==1.2.3 pillow six IPython;
+        sed -i "s/ENABLE_USER_SITE = .*/ENABLE_USER_SITE = False/g" /home/ubuntu/miniconda/envs/circleenv/lib/python2.7/site.py;
+      else
+        echo "Conda already set up.";
+      fi;
+    - ls -al /home/ubuntu/miniconda;
+    - ls -al /home/ubuntu/miniconda/bin;
+    - echo $PATH;
+    - which python;
+    - which pip;
+    - git clone https://github.com/sphinx-gallery/sphinx-gallery.git;
+    - cd sphinx-gallery && pip install -r requirements.txt && python setup.py develop;
+    - cd /home/ubuntu && git clone https://github.com/enthought/pyface.git && cd pyface && python setup.py develop
+    - pip install sphinx_bootstrap_theme PySurfer nilearn neo;
+    - cd /home/ubuntu/mne-python && python setup.py develop;
+    - mkdir -p ~/mne_data;
+    - make sample_data;
+    - make testing_data;
+    - make spm_data;
+    - make somato_data;
+    - cd /home/ubuntu/.mne && echo '{"MNE_DATASETS_BRAINSTORM_PATH":"/home/ubuntu/mne_data","MNE_DATASETS_EEGBCI_PATH":"/home/ubuntu/mne_data","MNE_DATASETS_MEGSIM_PATH":"/home/ubuntu/mne_data","MNE_DATASETS_SAMPLE_PATH":"/home/ubuntu/mne_data","MNE_DATASETS_SOMATO_PATH":"/home/ubuntu/mne_data","MNE_DATASETS_SPM_FACE_PATH":"/home/ubuntu/mne_data","MNE_DATASETS_TESTING_PATH":"/home/ubuntu/mne_data"}' > mne-python.json;
```

Comment:
You shouldn't need to do this because it should be written to the mne-python config file by the `make sample_data` above. If you're trying to get it to use `mne_data` when installing instead, this line needs to happen before all the `make` commands. Make sense?